src/seedu/addressbook/data/AddressBook.java

```diff
@@ -89,6 +94,14 @@ public void addPerson(Person toAdd) throws DuplicatePersonException {
     public void addTag(Tag toAdd) throws DuplicateTagException {
         allTags.add(toAdd);
     }
+    
+    public void addTagging(Tagging toAdd) {
+    	allTaggings.add(toAdd);
+    }
+    
+    public void deleteTagging(Tagging toDelete) {
+    	allTaggings.delete(toDelete);
+    }
```

Comment:
How to do you keep track of which tags are added and deleted? You need to print this information out when the program exits.